File-Formats/Presentation/FAQ.md

```diff
@@ -46,3 +46,56 @@ documentation: UG
    
    You can improve the quality of converted images by specifying the image resolution. Refer – [Converting PowerPoint presentation to Images](/file-formats/presentation/getting-started#converting-powerpoint-presentation-to-images)
 
+10. ## What are the NuGet packages to be installed to perform PowerPoint Presentation to PDF conversion in Linux OS?
```

Comment:
Please add this FAQ link in PPT to PDF conversion page

Reply:
Once FAQ MR merged. I will create separate MR for that